tests/views/test_authentication.py

```diff
@@ -5,12 +5,41 @@
 from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
 
+from jwt import get_unverified_header
+
+from pytest import skip
+
 from rest_framework import status
 
 from rest_framework_jwt.authentication import JSONWebTokenAuthentication
 from rest_framework_jwt.compat import has_set_cookie_samesite
 from rest_framework_jwt.settings import api_settings
 
+from sys import version_info
+
+try:
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives.asymmetric import rsa
+
+    # DO NOT copy key_size=512 for any real code!
+    srsa1 = rsa.generate_private_key(
+        public_exponent=65537,
+        key_size=512,
+        backend=default_backend()
+    )
+
+    srsa2 = rsa.generate_private_key(
+        public_exponent=65537,
+        key_size=512,
+        backend=default_backend()
+    )
+    prsa1 = srsa1.public_key()
+    prsa2 = srsa2.public_key()
+
+except:
```

Comment:
This is a known [anti-pattern in Python](https://realpython.com/the-most-diabolical-python-antipattern/). Which exceptions are expected to be raised here?

If these keys need to be reused, shouldn't they be placed in a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html)?

Reply:
thank you for helping out a python dummy :)